.rustfmt.toml

```diff
@@ -1 +0,0 @@
-wrap_comments = true
```

Comment:
Just curious, is this default now or you think it's not needed?

Reply:
It's not available in `stable` yet :(